### PR TITLE
PROJ-136

### PR DESCRIPTION
**Pull request by Emilo067**
### Description
This pull request includes the following changes related to the **Privacy Policy** and **Terms of Service** pages:

1. **Privacy Policy Page:**
   - Added new sections to provide users with clear information on how their data is collected, stored, and used.
   - Updated existing clauses to ensure compliance with the latest data protection regulations (e.g., GDPR, CCPA).
   - Improved readability and structure for better user experience.

2. **Terms of Service Page:**
   - Revised terms related to user rights and obligations when using the platform.
   - Added new clauses to address content ownership, intellectual property rights, and dispute resolution processes.
   - Simplified legal jargon to make the document more user-friendly and accessible.

### Changes
- New content sections added to Privacy Policy.
- Improved wording and structure for both Privacy Policy and Terms of Service.
- Ensured full compliance with recent legal standards.
  
### Testing
- Manually verified layout and formatting of the documents on the website.
- Ensured links and cross-references between sections are working correctly.

### Screenshots (if applicable)
terms of service
privacy policy

Please review the changes and provide feedback or approve the pull request.

### CODE REVIEW

File: src/features/auth/ui/privacyPolicy/privacyPolicy.tsx [hunk 1]
```diff
@@ -1,4 +1,6 @@
+import { PrivacyList } from '@/features/auth/ui/privacyPolicy/privacyList'
```
Comment (Lelik7777) [inline src/features/auth/ui/privacyPolicy/privacyPolicy.tsx]: ./privacyList

File: src/features/auth/ui/termsOfService/termsList.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import useTerms, { Term } from '@/features/auth/ui/termsOfService/useTerms'
```
Comment (Lelik7777) [inline src/features/auth/ui/termsOfService/termsList.tsx]: ./useTerms

File: src/features/auth/ui/termsOfService/termsList.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { Typography } from '@photo-fiesta/ui-lib'
+
+import styles from './termsOfService.module.scss'
+
+export const TermsList = () => {
```
Comment (Lelik7777) [inline src/features/auth/ui/termsOfService/termsList.tsx:21]: add js doc comments

File: src/features/auth/ui/termsOfService/termsOfService.tsx [hunk 4]
```diff
@@ -1,4 +1,6 @@
+import { TermsList } from '@/features/auth/ui/termsOfService/termsList'
```
Comment (Lelik7777) [inline src/features/auth/ui/termsOfService/termsOfService.tsx]: ./termList

File: src/features/auth/ui/termsOfService/useTerms.ts [hunk 5]
```diff
@@ -0,0 +1,71 @@
+import { useTranslation } from '@/shared/utils'
+
+export type Term = {
+  description: string
+  extra?: string[]
+  title: string
+}
+
+const useTerms = () => {
```
Comment (Lelik7777) [inline src/features/auth/ui/termsOfService/useTerms.ts:16]: add js doc comments

File: src/features/auth/ui/privacyPolicy/privacyPolicy.tsx (hunk 1 above)
Comment (Emilo067) [reply src/features/auth/ui/privacyPolicy/privacyPolicy.tsx]: исправил

File: src/features/auth/ui/termsOfService/termsList.tsx (hunk 2 above)
Comment (Emilo067) [reply src/features/auth/ui/termsOfService/termsList.tsx]: исправил

File: src/features/auth/ui/termsOfService/termsOfService.tsx (hunk 4 above)
Comment (Emilo067) [reply src/features/auth/ui/termsOfService/termsOfService.tsx]: исправил

File: src/features/auth/ui/termsOfService/termsList.tsx (hunk 2 above)
Comment (Emilo067) [reply src/features/auth/ui/termsOfService/termsList.tsx]: исправил

File: src/features/auth/ui/termsOfService/termsList.tsx (hunk 3 above)
Comment (Emilo067) [reply src/features/auth/ui/termsOfService/termsList.tsx:21]: исправил

File: src/features/auth/ui/termsOfService/termsOfService.tsx (hunk 4 above)
Comment (Emilo067) [reply src/features/auth/ui/termsOfService/termsOfService.tsx]: исправил

File: src/features/auth/ui/termsOfService/useTerms.ts (hunk 5 above)
Comment (Emilo067) [reply src/features/auth/ui/termsOfService/useTerms.ts:16]: исправил

**Comment by Lelik7777** [comment]
run build production

**Pull request merged**